mmgen/__init__.py

```diff
@@ -17,7 +17,7 @@ def digit_version(version_str):
 
 
 mmcv_minimum_version = '1.3.0'
-mmcv_maximum_version = '1.5.0'
+mmcv_maximum_version = '1.6.0'
```

Comment:
'1.5.1' ?

Reply:
We can directly define a higher version to avoid fixing this frequently. 
Refers to https://github.com/open-mmlab/mmsegmentation/blob/4f394b06743860543a0e7135f7e7496bdeb79764/mmseg/__init__.py#L10